### ISSUE TITLE
Stubs are out of date

### ISSUE THREAD

**Issue by simonwhitaker** [sbdchd/django-types#255]
The stubs in this repo are considerably out of date with upstream. This can be seen clearly in https://github.com/sbdchd/django-types/blob/main/django-stubs/utils/deprecation.pyi, where I see this snippet:

```py
class RemovedInDjango30Warning(PendingDeprecationWarning): ...
class RemovedInDjango31Warning(PendingDeprecationWarning): ...
class RemovedInDjango40Warning(PendingDeprecationWarning): ...
class RemovedInNextVersionWarning(DeprecationWarning): ...
```

Compare with the [head of upstream](https://github.com/django/django/blob/main/django/utils/deprecation.py):

```py
class RemovedInDjango60Warning(DeprecationWarning):
    pass


class RemovedInDjango61Warning(PendingDeprecationWarning):
    pass


RemovedInNextVersionWarning = RemovedInDjango60Warning
RemovedAfterNextVersionWarning = RemovedInDjango61Warning
```

This is a problem, because Microsoft's Pylance extension for VSCode [uses this repo](https://github.com/microsoft/pylance-release/issues/6029#issuecomment-2176515647) for the source of its bundled Django stubs. As a result, if I enable type checking in a modern Django project (Django 5.0.x), I get a lot of erroneous type errors, such as this:

 
<img width="1022" alt="Screenshot 2024-06-18 at 5 06 14 PM" src="https://github.com/sbdchd/django-types/assets/116432/3aef3973-9d11-4b14-8d33-e0061a152aa6">

I reported this as an issue in Pylance (https://github.com/microsoft/pylance-release/issues/6029) and they directed me to you.

**Comment by ldeluigi** [comment]
Probably related to:
https://github.com/sbdchd/django-types/issues/255

**Comment by sbdchd** [comment]
yeah stubs will fall behind, I only added things that I've needed when I've need them rather than trying to keep 1:1 with Django

for instance, in my django project I don't use admin so there wasn't any admin definitions, but someone else needed them so they added them to this project
